scripts/test/Muon/elemental_analysis/periodic_table_item_test.py

```diff
@@ -0,0 +1,72 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+from __future__ import print_function, absolute_import
+
+
+import unittest
+
+from mantid.py3compat import mock
+
+import Muon.GUI.ElementalAnalysis.PeriodicTable.periodic_table as periodic_table
+from Muon.GUI.ElementalAnalysis.PeriodicTable.periodic_table import PeriodicTableItem,\
+    ColoredPeriodicTableItem
+
+
+class PeriodicTableItemTest(unittest.TestCase):
+    def setUp(self):
+        self.element = PeriodicTableItem("Ti", 22, 4, 4, "titanium", 47.9000, "transition metal")
+
+    def test_that_list_elements_contains_all(self):
+        for i in range(len(periodic_table._elements)-1):
```

Comment:
I dont understand what this code is doing

Reply:
It checks that all elements in the periodic table are provided with all the required fields:
Symbol, Atomic Number, Col, Row, Name, Mass, Subcategory
I added a comment to explain this